android/ot-sdk-wrapper/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -496,6 +498,14 @@ public void sendSignal(SignalInfo signalInfo) {
             internalSendSignal(signalInfo);
         }
     }
+    /**
+     * Whether the local MediaType is enabled (<code>true</code>) or not (
+     * <code>false</code>).
+     */
+    public boolean getLocalMediaEnabled(MediaType type) {
```

Comment:
We have the same method but with other name: public boolean isLocalMediaEnabled(MediaType type) {
        return (mPublisher != null) &&
                (type == MediaType.VIDEO ? mPublisher.getPublishVideo() : mPublisher.getPublishAudio());
    }